### ISSUE TITLE
unlike demo scroll not visible except on hover

### ISSUE THREAD

**Issue by Egzscure** [MaTeMaTuK/gantt-task-react#92]
In the demo i can always see the scrollbars (vertical and horizontal) however in the component it doesnt show the scrollbars unless i hover my mouse. 

I managed to make the horizontal scroll bar like the demo, but i cant get the vertical one to stay visible like in the demo, im using the latest version.

Why is that?

**Comment by MaTeMaTuK** [comment]
Hi @Egzscure,
Do you use Chrome on PC ?

**Comment by Egzscure** [comment]
Yes, i managed to make it work with some css tweaking but its not the same as the demo

**Comment by MaTeMaTuK** [comment]
Could you please try to reproduce on sandbox ? 
https://codesandbox.io/s/vigilant-star-fxy36r?file=/src/App.tsx
